apps/cloud_composer/src/project_model.cpp

```diff
@@ -53,6 +53,7 @@ pcl::cloud_composer::ProjectModel::ProjectModel (QObject* parent)
 }
 
 pcl::cloud_composer::ProjectModel::ProjectModel (const ProjectModel&)
+: QStandardItemModel ()
```

Comment:
Call copy constructor and not default constructor.

Reply:
AFAIR I tried and it did not compile.

Reply:
Well I started a build of #3076 on my VM. Will check output, if it is ready (seems ccache is invalided due to the other accepted PRs, so it will need a while). Maybe I check this this night or tomorrow.

Reply:
```
error: ‘QStandardItemModel::QStandardItemModel(const QStandardItemModel&)’ is private within this context
 : QStandardItemModel ( to_copy)
```

Reply:
Sounds like base class is private, but it isn't. I wait for output of clang, maybe error message is more helpful there.

Reply:
Uff. If we're deriving from a non copyable class, this case needs more attention. It might be worth to split it to a new PR.

Reply:
Just to mention: As far as I can see we should inherit from `QAbstractItemModel`, because this is copyable.

Reply:
Deriving `ProjectModel` from `QAbstractItemModel` instead of `QStandardItemModel` does not work.

Reply:
Why not?

Reply:
There is a flurry of errors, here is one for example:

```cpp
../apps/cloud_composer/src/point_selectors/selected_trackball_interactor_style.cpp:32:35: error: ‘class pcl::cloud_composer::ProjectModel’ has no member named ‘itemFromIndex’
     QStandardItem* item = model_->itemFromIndex (index);
```

Reply:
Of course you need to implement some methods of `QStandardItemModel` again, if you need them ;).

Reply:
OK, this is not a quick fix then.

Reply:
Indeed. You could open a issue, in case someone has boredom and want to do sth ;).